frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/enterprise.unit.spec.tsx

```diff
@@ -1,234 +0,0 @@
-import "metabase/plugins/builtin";
-import userEvent from "@testing-library/user-event";
-import fetchMock from "fetch-mock";
-
-import { setupGroupsEndpoint } from "__support__/server-mocks";
-import { screen, within } from "__support__/ui";
-import {
-  createMockGroup,
-  createMockSettingDefinition,
-  createMockSettings,
-  createMockTokenFeatures,
-} from "metabase-types/api/mocks";
-
-import type { SetupOpts } from "./setup";
-import { EMAIL_URL, setup } from "./setup";
-
-const setupEnterprise = async (opts?: SetupOpts) => {
-  await setup({ ...opts, hasEnterprisePlugins: true });
-};
-
-describe("SettingsEditor", () => {
-  it("shows notify admin of new users provisioned options", async () => {
-    fetchMock.get("path:/api/ee/scim/api_key", 404);
-
-    await setupEnterprise({
-      initialRoute: "/admin/settings/authentication/user-provisioning",
-      settings: [
-        createMockSettingDefinition({ key: "saml-enabled", value: true }),
-      ],
-      settingValues: createMockSettings({ "saml-enabled": true }),
-      tokenFeatures: createMockTokenFeatures({ scim: true, sso_saml: true }),
-    });
-
-    expect(
-      await screen.findByText(
-        "Notify admins of new users provisioned from SSO",
-      ),
-    ).toBeInTheDocument();
-  });
-
-  it("should not allow to configure the origin and SameSite cookie for interactive embedding", async () => {
```

Comment:
some tests don't even make sense in the new settings - this is a test that a setting does not exist on a page. maybe there was a risk of it before, but it doesn't make sense anymore. It's like testing that the move collection modal doesn't exist on the login page 🤷